lib/matplotlib/axes/_base.py

```diff
@@ -3700,32 +3700,26 @@ def yaxis_date(self, tz=None):
 
     def format_xdata(self, x):
         """
-        Return *x* string formatted.  This function will use the attribute
-        self.fmt_xdata if it is callable, else will fall back on the xaxis
-        major formatter
+        Return *x* formatted as an x-value.
+
+        This function will use the :attr:`fmt_xdata` attribute if it is not
```

Comment:
‘.fmt_xdata’ should be sufficient.

On a phone so formatting is a bit difficult 🙄

Reply:
done